### PR TITLE
Add docs via nbsphinx

### PR DESCRIPTION
**Pull request by ocefpaf**
I'm almost done, just checking if this build locally.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -44,3 +44,19 @@ script:
...
+      jupyter nbconvert --to notebook --execute notebooks/demo.ipynb --output=demo-output.ipynb
+      cp notebooks/demo-output.ipynb docs/source/
+      pushd docs
+      make clean html linkcheck
+      popd
+      # if [[ -z "$TRAVIS_TAG" ]]; then
+      #   python -m doctr deploy --build-tags --key-path github_deploy_key.enc --built-docs docs/build/html dev
+      # else
+      #   python -m doctr deploy --build-tags --key-path github_deploy_key.enc --built-docs docs/build/html "version-$TRAVIS_TAG"
+      #   python -m doctr deploy --build-tags --key-path github_deploy_key.enc --built-docs docs/build/html .
```
Comment (ocefpaf) [inline .travis.yml:60]: We are not publishing the docs yet but that requires GH permission on the OHW organization. I want to move this project to the IOOS organization later so we can do that.

File: gliderpy/fetchers.py [hunk 2]
```diff
@@ -90,25 +90,20 @@ def plot_track(self):
...
+
         """
         Makes a scatter plot of depth vs time coloured by a user defined variable
         :param var: variable to colour the scatter plot
         :return: figure, axes
         """
-        
+
         idx = self.to_pandas()
-        cmap = Haline_20.mpl_colormap
```
Comment (ocefpaf) [inline gliderpy/fetchers.py:102]: @LindsayRAbrams I removed the salinity colormap for now and opened #21 so we can handle this more gracefully later. Let's leave the default cmap for the presentation.

**Comment by ocefpaf** [comment]
@LindsayRAbrams can you take a quick look? If you are OK with this PR merge it and I won't touch the notebook or the main module so you can work on the plotting stuff.

**Comment by LindsayRAbrams** [comment]
Looks good to me! Are you good with me merging this PR once the checks are complete?

**Comment by ocefpaf** [comment]
> Looks good to me! Are you good with me merging this PR once the checks are complete?

Ideally we should merge PRs only when the CIs pass but pre-commit is failing due to the lack of docstring coverage and [a typo](https://github.com/oceanhackweek/ohw20-proj-glide-data-fetcher/pull/22/checks?check_run_id=985317284#step:6:122), so no big deal.

And Travis-CI is failing b/c I commented out the publication part on purpose. So in this case we are OK merging with a failing CI.

**Comment by ocefpaf** [comment]
Let's get this merged to you can work on #20

**Pull request merged**